src/Symfony/Component/Intl/Countries.php

```diff
@@ -35,7 +37,13 @@ final class Countries extends ResourceBundle
      */
     public static function getCountryCodes(): array
     {
-        return self::readEntry(['Regions'], 'meta');
+        self::ensureOptionalUserAssignedResolved();
+
+        if (!self::$optionalUserAssigned) {
```

Comment:
If the method returns the value, the code is shorter.
```suggestion
        if (!self::optionalUserAssigned()) {
```